app/views/data_centers/index.html.erb

```diff
@@ -1,35 +1,25 @@
 <h1 class="page-header"><%= params[:network].capitalize %> Data Centers</h1>
-
+<div class=" mb-4 col-sm-4" role="toolbar" aria-label="filter"> 
+  <%= link_to "Data Centers", \
+              data_centers_url(network: params[:network], sort_by: :data_center), \
+              class: "btn #{[nil, 'data_center'].include?(@sort_by) ? 'btn-primary' : 'btn-outline-primary'}", \
```

Comment:
Please do not use ```btn-outline-primary``` class, it's not fully styled in app (has some bootstrap left-overs that we don't want). Use ```btn-secondary``` instead.

Reply:
Also, please add ```mr-1``` class to all buttons